seqr/views/apis/project_categories_api_tests.py

```diff
@@ -4,15 +4,21 @@
 from seqr.views.apis.project_categories_api import update_project_categories_handler
 from seqr.views.utils.test_utils import _check_login
 
+PROJECT_GUID = 'R0001_1kg'
+
 
 class ProjectCategoriesAPITest(TestCase):
     fixtures = ['users', '1kg_project']
 
     def test_project_categories_api(self):
-        url = reverse(update_project_categories_handler, args=['R0001_1kg'])
+        url = reverse(update_project_categories_handler, args=[PROJECT_GUID])
         _check_login(self, url)
 
         response = self.client.post(url, content_type='application/json', data=json.dumps({
-            'categories': []
+            'categories': ['PC000002_categry_with_unicde', 'PC000003_categry_with_unicde']
         }))
         self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+
+        self.assertListEqual(response_json.keys(), ['projectsByGuid', 'projectCategoriesByGuid'])
+        self.assertListEqual(response_json['projectsByGuid'].keys(), [PROJECT_GUID])
```

Comment:
Can you test that you have the expected project categories in the response as well? And can you check that the project json now has the correct project category guids in the response

Reply:
My comment here still stands - can you please test that the `projectCategoryGuids` in the project json are correct, both here and after you have added new categories below?

Reply:
Added the content of "projectCategoryGuids" test for both api call responses.

Reply:
Yes, but what I was saying is you need to test the value of the `projectCategoryGuids` field inside the PROJECT item. As in, inside the json in `projectsByGuid`

Reply:
Do you mean I should query the database rather than just checking the responses?

Reply:
Added examining the responses against the data in the database after the update_project_categories_handler API call.

Reply:
While you are now doing a great job checking the content of `response_json['projectCategoriesByGuid']`, there are still no tests for `response_json['projectsByGuid'][PROJECT_GUID]['projectCategoryGuids']` as I requested

Reply:
I got your point finally. Sorry for the delay. I kept taking response_json['projectsByGuid'][PROJECT_GUID]['projectCategoryGuids'] as response_json['projectCategoriesByGuid'] by mistake.